app/models/compute_resources/foreman/model/openstack.rb

```diff
@@ -35,6 +36,22 @@ def tenant=(name)
       attrs[:tenant] = name
     end
 
+    def allow_ext_net?
+      !(attrs[:allow_ext_net] == 0)
+    end
+
+    def allow_ext_net
+      attrs[:allow_ext_net]
+    end
+
+    def allow_ext_net=(enabled)
+      if ['true', true, '1', 1].include?(enabled)
```

Comment:
@dLobatog I am validating 0 and 1 because this is what the `checkbox_f` method returns (always string, no boolean, so I just left the default 0/1).

Also, this is not passed to Openstack - we're using this value to decide whether to exclude the external networks on Foreman's side (see the `internal_networks` method)